docs/rules/no-test-module-for.md

```diff
@@ -0,0 +1,23 @@
+# Use `module` instead of `moduleFor` (no-test-module-for)
+
+`moduleForComponent`, `moduleFor`, `moduleForAcceptance`, etc have been deprecated and there are codemods to help migrate.
```

Comment:
Can you add a bullet-point list of alternatives / replacements / techniques to migrate away from the `moduleFor` function?

Reply:
maybe addressed here? 871251aa9bb2a92bb33568716d3bf18699b32d08 ?

I don't want to duplicate suggestions covered in codemods' readmes

Reply:
Yeah, no need to duplicate all the details, but I'd like to make sure our documentation includes the key points so it can stand on its own.